modules/food.py

```diff
@@ -0,0 +1,39 @@
+#1/usr/bin/env/python
+"""
+food.py - Jenni Food Module
+by afuhrtrumpet
+
+More info:
+ * Jenni: https://github.com/myano/jenni/
+ * Phenny: http://inamidst.com/phenny/
+"""
+from yelpapi import YelpAPI
+import random
+
+def food(jenni, input):
+	yelp_api = YelpAPI(jenni.config.yelp_api_credentials['consumer_key'], jenni.config.yelp_api_credentials['consumer_secret'], jenni.config.yelp_api_credentials['token'], jenni.config.yelp_api_credentials['token_secret'])
+
+	location = input.group(2)
+	if not location:
+		location = '43210'
```

Comment:
You should force the user to enter location instead of using an arbitrary one by saying something like:
```
jenni.say("Perhaps you meant .food 90210?")
```